libs/domain/checkout/address/stories/static.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { resolve } from '@spryker-oryx/di';
+import { AddressService } from '@spryker-oryx/user';
+import {
+  MockAddressService,
+  MockAddressType,
+  mockNormalizedAddresses,
+} from '@spryker-oryx/user/mocks';
+import { Meta, Story } from '@storybook/web-components';
+import { html, TemplateResult } from 'lit';
+import { storybookPrefix } from '../../.constants';
+
+export default {
+  title: `${storybookPrefix}/Address`,
+} as Meta;
+
+const service = resolve<MockAddressService>(AddressService);
+service.changeMockAddressType(MockAddressType.ThreeWithDefaults);
+
+const Template: Story = (): TemplateResult => {
+  return html`
+    <h3>(multiline)</h3>
```

Comment:
Do we need brackets it each title? I don't remember us using something like this in other stories

Reply:
We do not, will drop it as this is static anyway